CMP1903M Object Oriented Programming A01 2223/CMP1903M A01 2223/CMP1903M A01 2223/Testing.cs

```diff
@@ -9,19 +9,19 @@ namespace CMP1903M_A01_2223
     class Testing
     {
         public void Test()
-        {
-            int[] cardArray = { };
-            
+        {   
             //Instantiates objects
-            Card c1 = new Card();
             Pack p1 = new Pack();
 
-            int[] cards = p1.cardPack.ToArray();
-
             //Uses the methods for the Pack class
-            p1.shuffleCardPack(1, cards);
-            p1.deal(cards);
-            p1.dealCard(52, cards);
+            Pack.shuffleCardPack(1);
+            Console.WriteLine("Test 1 successful");

```

Comment:
Not sure how this could be implemented in a simple fashion. But as an Idea you could have it so that the methods called return a boolean value if they succeed thus allowing you to create an accurate message if the method worked or not.

Note: That would be tricky to implement and probably not worth it just throwing it out there as an Idea.